src/library/Uno.Material/Styles/Controls/v2/PipsPager.Base.xaml

```diff
@@ -61,7 +61,7 @@
 			<StaticResource x:Key="MaterialPipsPagerNavigationButtonVisualStateEllipseFillPointerOver" ResourceKey="PrimaryHoverBrush" />
 			<StaticResource x:Key="MaterialPipsPagerNavigationButtonVisualStateEllipseFillPressed" ResourceKey="PrimaryPressedBrush" />
 			<StaticResource x:Key="MaterialPipsPagerNavigationButtonVisualStateEllipseFillFocused" ResourceKey="PrimaryFocusedBrush" />
-			<StaticResource x:Key="MaterialPipsPagerNavigationButtonVisualStateEllipseFillUnfocused" ResourceKey="PrimaryUnfocusedBrush" />
+			<StaticResource x:Key="MaterialPipsPagerNavigationButtonVisualStateEllipseFillUnfocused" ResourceKey="SystemControlTransparentBrush" />
```

Comment:
I think we need a doc change in https://github.com/unoplatform/Uno.Themes/blob/master/doc/styles/PipsPager.md